Sources/RunnerLib/VersionChecker.swift

```diff
@@ -0,0 +1,50 @@
+import DangerShellExecutor
+import Logger
+import Version
+import Foundation
+
+public enum VersionChecker {
+    public static func checkForUpdate(current currentVersionString: String) {
```

Comment:
Can inject ShellExecutor and Logger (which anyway can be needed), so that we can also have some tests for this logic?

Reply:
Fixed `VersionChecker` enum to class, which make it injectable 😄.